public/js/users/index-member.js

```diff
@@ -1,17 +1,85 @@
 const { banner, validSpot } = require('../utils');
 
-const apiEndPoint = '/users/spot';
-const editButtonClass = '.UserIndex-editSpot';
-const confirmEditTextBoxButtonClass = 'UserIndex-confirmEditTextButton';
+const apiEndPoint = '/users';
+const editPartButtonClass = '.UserIndex-editPart';
+const editSpotButtonClass = '.UserIndex-editSpot';
+const cancelEditPartButtonClass = 'UserIndex-cancelEditPartButton';
 const cancelEditTextBoxButtonClass = 'UserIndex-cancelEditTextBoxButton';
+const confirmEditPartButtonClass = 'UserIndex-confirmEditPartButton';
+const confirmEditTextBoxButtonClass = 'UserIndex-confirmEditTextButton';
+const instrumentPartMap = {
+  Baritone: ['First'],
+  Mellophone: ['First', 'Second'],
+  Percussion: ['Bass', 'Cymbals', 'Snare', 'Tenor'],
+  Sousaphone: ['First'],
+  Trombone: ['First', 'Second', 'Bass'],
+  Trumpet: ['Efer', 'Solo', 'First', 'Second', 'Flugel']
+};
+const selectClass = 'UserIndex-partSelect';
 const textAreaClass = 'UserIndex-textarea';
 const userIndexItemClass = 'UserIndex-userItem';
 
-$(editButtonClass).on('click', (e) => {
-  clickEventFunction(e);
+$(editSpotButtonClass).on('click', (e) => {
+  editSpotClickEventFunction(e);
 });
 
-const clickEventFunction = (e) => {
+$(editPartButtonClass).on('click', (e) => {
+  editPartClickEventFunction(e);
+});
+
+const editPartClickEventFunction = (e) => {
```

Comment:
If another button needs added to change stuff, strongly consider refactoring this. There's some reusable code here